pythonfmu/pythonfmu-export/src/pythonfmu/PySlaveInstance.cpp

```diff
@@ -150,7 +210,7 @@ void PySlaveInstance::initialize(PyGILState_STATE gilState)
     Py_XDECREF(pMessages_);
 
     PyObject* args = PyTuple_New(0);
-    PyObject* kwargs = Py_BuildValue("{ss,ss,sn,si}",
+    PyObject* kwargs = Py_BuildValue("{ss,ss,sL,si}",
```

Comment:
`sL` -> `sn` was changed recently. Is this intended or merge artifact?
See https://github.com/NTNU-IHB/PythonFMU/pull/181

Reply:
Probably a merge artifact, I'll revert!